app/cmd/restore_to_file.go

```diff
@@ -93,7 +94,7 @@ func restore(url string, concurrentLimit int) error {
 	bar := pb.StartNew(100)
 	periodicChecker := time.NewTicker(PeriodicRefreshIntervalInSeconds * time.Second)
 
-	progressKeyName := requestedBackupName + util.RandomID(8)
+	progressKeyName := requestedBackupName + util.RandomID()
```

Comment:
Move `RandomID` function to go-common-libs.
nit: I'm fine with the parameter, randomIDLenth` which makes the function more generalized.

Reply:
Fixed, thanks!